pylint/pyreverse/utils.py

```diff
@@ -292,7 +293,34 @@ def check_graphviz_availability():
     if shutil.which("dot") is None:
         print(
             "The requested output format is currently not available.\n"
-            "Please install 'Graphviz' to have other output formats "
-            "than 'dot' or 'vcg'."
+            "Please install 'Graphviz' to have image output formats."
```

Comment:
The message was outdated as we now support other output formats as well. Reiterating the directly supported output formats here is not necessary I think, so I kept it more general.

Reply:
```suggestion
            "'Graphviz' need to be installed for your chosen output format."
```